app/src/main/java/org/inspire/breath/validateDiarrhoea.java

```diff
@@ -0,0 +1,71 @@
+package org.inspire.breath;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.TextView;
+import android.widget.Button;
+
+public class validateDiarrhoea extends AppCompatActivity {
+
+    Button answerYes, answerNo;
+    TextView question;
+    private Questions mQuestions = new Questions();
+    private String mAnswer;
+    private int mQuestionsLength = mQuestions.mQuestions.length;
+    private int questionNum = 0;
+    @Override
+
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_validate_diarrhoea);
+
+        answerYes = (Button) findViewById(R.id.buttonYes);
+        answerNo = (Button) findViewById(R.id.buttonNo);
+
+        question = (TextView) findViewById(R.id.question);
+        question.setText(mQuestions.getQuestion(questionNum));
+        answerYes.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                if (questionNum<2) {
+                    questionNum++;
+                    question.setText(mQuestions.getQuestion(questionNum));
+                }
+                else if (questionNum>=2){
+                    question.setText(mQuestions.getAnswer(1));
+                    answerYes.setVisibility(View.GONE);
+                    answerNo.setVisibility(View.GONE);
+                }
+
+
+            }
+        });
+
+        answerNo.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
```

Comment:
Can this be reduced to just the inner 3 lines using the index? The if/else checks are redundant